runtime.go

```diff
@@ -75,8 +75,8 @@ type State struct {
 
 var builtinsLock sync.Mutex
 
-// newOPARuntime creates a new OPA Runtime.
-func newOPARuntime(ctx context.Context, log *zerolog.Logger, cfg *Config, opts ...Option) (*Runtime, func(), error) {
+// New creates a new OPA Runtime.
+func New(ctx context.Context, log *zerolog.Logger, cfg *Config, opts ...Option) (*Runtime, error) {
```

Comment:
Would it be better to pass the logger by context, so we can remove the log param?

Reply:
That's a good idea. I'll make the change.